savant/utils/image.py

```diff
@@ -0,0 +1,528 @@
+import math
+from typing import Union, Tuple, Optional
+
+import cv2
+import numpy as np
+
+from savant.meta.bbox import BBox, RBBox
+
+
+class CPUImage:
+    def __init__(self, image: np.ndarray):
+        """Image data container.
+
+        :param image: image data
+        """
+        self._np_image = image
+
+    @property
+    def np_array(self) -> np.ndarray:
+        """Returns numpy image.
+
+        :return: numpy image
+        """
+        return self._np_image
+
+    @property
+    def to_gpu(self) -> cv2.cuda.GpuMat:
+        """Returns GPU image.
+
+        :return: GPU image
+        """
+        image_gpu = cv2.cuda.GpuMat(self._np_image.shape)
+        image_gpu.upload(self._np_image)
+        return image_gpu
```

Comment:
```suggestion
        return cv2.cuda.GpuMat(self._np_image)
```

Just a small simplification